bot.py

```diff
@@ -175,4 +175,34 @@ async def watchlist_join(interaction: nextcord.Interaction, watchlist_name):
         response = f"Watchlist named {watchlist_name} does not exist"
     await interaction.response.send_message(response)
 
+@bot.slash_command(guild_ids=[GUILD_ID], name="watchlist_leave", description="leave from a joined watchlist")
+async def watchlist_delete(interaction: nextcord.Interaction, watchlist_name):
```

Comment:
should this still be called watchlist_delete?

Reply:
Haha nice find, can't believe I made the same mistake again.